packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.ct.tsx

```diff
@@ -93,6 +103,30 @@ test.describe("Empty screenshots", () => {
   });
 });
 
+test.describe("Truncated options screenshots", () => {
+  executeMatrixScreenshotTest({
+    name: "Select (truncated)",
+    columns: DENSITIES,
+    rows: ["ellipsis", "multiline"],
+    disabledAccessibilityRules: DISABLED_ACCESSIBILITY_RULES,
+    component: (column, row) => (
+      <OnyxSelect
+        label="Label"
+        listLabel="List label"
+        options={
+          row === "ellipsis" ? MOCK_LONG_LABELED_OPTIONS : MOCK_MULTILINE_LONG_LABELED_OPTIONS
+        }
+        density={column}
+      />
+    ),
+    beforeScreenshot: async (component) => {
+      await openFlyout(component);
+      const option = component.getByText(`Long labeled option 1 `.repeat(4));
```

Comment:
I'm referring to here :)

```suggestion
      const option = component.getByText(`Long labeled option 1 `);
```